code/modules/martial_arts/combos/judo/wheelthrow.dm

```diff
@@ -0,0 +1,19 @@
+/datum/martial_combo/judo/wheelthrow
+	name = "Wheel Throw"
+	steps = list(MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_HARM)
+	explaination_text = "From an armbar, flip your opponent over your shoulder, slamming them onto the floor, leaving them stunned."
+	combo_text_override = "Grab, Disarm, Harm"
+
+/datum/martial_combo/judo/wheelthrow/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(!IS_HORIZONTAL(target) || !(target.IsImmobilized() || user.IsKnockedDown()))
+		return MARTIAL_COMBO_FAIL
+	target.visible_message("<span class='warning'>[user] raises [target] over [user.p_their()] shoulder, and slams them into the ground!</span>", \
+						"<span class='userdanger'>[user] !</span>")
```

Comment:
I think you forgot a message here

Reply:
Thanks for the review! I've addressed everything in the latest commit!